src/bootstrap/Cargo.toml

```diff
@@ -7,6 +7,7 @@ default-run = "bootstrap"
 
 [features]
 build-metrics = ["sysinfo"]
+logging = ["dep:tracing", "dep:tracing-subscriber", "dep:tracing-tree"]
```

Comment:
I think the name `logging` is too generic. I would call it `tracing` and use it with `BOOTSTRAP_TRACING`.

Reply:
Sounds reasonable, I'll change it over.

Reply:
Changed cargo feature to `tracing` and env var to `BOOTSTRAP_TRACING`.